Sources/MongoMobile/MongoMobile.swift

```diff
@@ -1,124 +1,137 @@
+import Foundation
 import mongo_embedded
```

Comment:
[q] Should this also have `import MongoSwift`?

Reply:
I tried, but MongoMobile won't build within another project (such as TestMongoMobile) if it's there

Reply:
Yeah, since MongoSwift is being vendored in it no longer exists in that package name. This is an unfortunate sideeffect of not being able to properly set up the dependency between the libraries.